features/filesystem/littlefs/LittleFileSystem.cpp

```diff
@@ -26,78 +26,100 @@ extern "C" {
 static int lfs_toerror(int err)
 {
     switch (err) {
-        case LFS_ERR_OK:        return 0;
-        case LFS_ERR_IO:        return -EIO;
-        case LFS_ERR_NOENT:     return -ENOENT;
-        case LFS_ERR_EXIST:     return -EEXIST;
-        case LFS_ERR_NOTDIR:    return -ENOTDIR;
-        case LFS_ERR_ISDIR:     return -EISDIR;
-        case LFS_ERR_INVAL:     return -EINVAL;
-        case LFS_ERR_NOSPC:     return -ENOSPC;
-        case LFS_ERR_NOMEM:     return -ENOMEM;
-        default:                return err;
+        case LFS_ERR_OK:
+            return 0;
+        case LFS_ERR_IO:
+            return -EIO;
+        case LFS_ERR_NOENT:
+            return -ENOENT;
+        case LFS_ERR_EXIST:
+            return -EEXIST;
+        case LFS_ERR_NOTDIR:
+            return -ENOTDIR;
+        case LFS_ERR_ISDIR:
+            return -EISDIR;
+        case LFS_ERR_INVAL:
+            return -EINVAL;
+        case LFS_ERR_NOSPC:
+            return -ENOSPC;
+        case LFS_ERR_NOMEM:
+            return -ENOMEM;
+        default:
+            return err;
```

Comment:
Do we have a stance on single line switch statements? These are pretty common in embedded programming and more difficult to read/edit in this format.

Reply:
One line one statement.

Reply:
Formally, that's a *labelled-statement* occupying two lines. Squish them together and you'd have one line one *labelled-statement*.

FWIW, I'm might be inclined to squish in those circumstances, but it's a very special case for exactly 1 label per 1 statement. Forcing the line break feels consistent with the logic for "single-line conditional statements are braced as if multiline".

Reply:
Ok, sounds like I'm in the minority so let's keep it as is